packages/apollo-server-core/src/requestPipeline.ts

```diff
@@ -484,7 +472,9 @@ export async function processGraphQLRequest<TContext>(
         // XXX Nothing guarantees that the only errors thrown or returned
         // in result.errors are GraphQLErrors, even though other code
         // (eg apollo-engine-reporting) assumes that.
-        return await config.executor(requestContext);
+        return await config.executor(
+          requestContext as GraphQLRequestContextExecutionDidStart<TContext>,
```

Comment:
Is the cast required here? I'm curious since it's typed the same coming into the function, so I think it's already known to be of type `GraphQLRequestContextExecutionDidStart`.

Reply:
Good catch; no longer required!  However, that wasn't the case prior me sorting out the omission of the very-much-present `queryHash` in my (2nd/follow-up) commit in this PR: d1b7a822e68245be5aebacd55437d226230745d8.

Reply:
Fixed by d4f9e6541edc1073552834195a47d1349d3f663f